src/libsystemd/sd-id128/sd-id128.c

```diff
@@ -214,8 +214,10 @@ static int get_invocation_from_keyring(sd_id128_t *ret) {
 
         key = request_key("user", "invocation_id", NULL, 0);
         if (key == -1) {
```

Comment:
turn this into < 0 check to match our coding style?

Reply:
hmm, not sure this is really the right approach in this case. keyring api knows various specific negative pseudo-key serials it takes in various calls. i.e. KEY_SPEC_USER_KEYRING is -4 and so on.

Now, I think request_key() would never return any of these special values (at least I hope), but only regular, positive serials, but i don't know for sure. Hence it felt better to just rely on whatever the man page says (which is: errors are -1), instead of assuming that KEY_SPEC_USER_KEYRING can never be returned here, even though in most other calls that's a fully valid key serial number...

Reply:
OK. Feels free to ignore then (but note we have one callsite of request_key() doing < 0 check, which shall be fixed later)